adafruit_imageload/pnm/__init__.py

```diff
@@ -1,24 +1,7 @@
-# The MIT License (MIT)
+# SPDX-FileCopyrightText: 2018 Scott Shawcroft for Adafruit Industries
```

Comment:
In this file, and some others there are more authors listed below. I'll work on pushing these changes and wrapping this one up later tonight.